apps/demo/src/app/cart/pages/cart-view/cart-view.component.scss

```diff
@@ -0,0 +1,13 @@
+.demo-cart-view {
+	&__cart {
+		padding: 50px 25px;
+
+		@include breakpoint(big-tablet) {
+			padding: 75px 50px;
+		}
+
+		@include breakpoint(laptop) {
+			padding: 75px 0;
+		}
+	}
+}
```

Comment:
These styles need to be included in the component. They aren't being applied right now.